src/main/java/org/broadinstitute/hellbender/utils/param/ParamUtils.java

```diff
@@ -128,4 +169,58 @@ public static double logb(final double val, final double base) {
     public static double log2(final double val) {
         return Math.log(val)/Math.log(2.0);
     }
+
+    /**
+     *  Writes a list of strings to the given filename.  The file will be overwritten if it exists.
+     *
+     * @param stringList Strings that will be written (one to a line)
+     * @param outputFile File to write
+     */
+    public static void writeStringListToFile(final List<String> stringList, final File outputFile) {
```

Comment:
How about call this `writeStringsToFile` and take any `Iterable<String>` as input?